commons/src/main/java/org/sonarsource/analyzer/commons/RuleMetadataLoader.java

```diff
@@ -59,6 +63,11 @@ public RuleMetadataLoader(String resourceFolder, String defaultProfilePath) {
     this(resourceFolder, BuiltInQualityProfileJsonLoader.loadActiveKeysFromJsonProfile(defaultProfilePath));
   }
 
+  public RuleMetadataLoader(String resourceFolder, String defaultProfilePath, SonarRuntime sonarRuntime) {
+    this(resourceFolder, BuiltInQualityProfileJsonLoader.loadActiveKeysFromJsonProfile(defaultProfilePath));
```

Comment:
you don't need to compute `BuiltInQualityProfileJsonLoader.loadActiveKeysFromJsonProfile(defaultProfilePath)` as you can call previous constructor (and avoid duplication this way)